content/app_competition_rules.tex

```diff
@@ -47,18 +47,32 @@ \subsection{Qualification Procedure and Code Usage}
 
 \subsection{Game Structure}
 
-The clock stops during stoppages of play (such as ready and set state after goals) from the quarter-finals onward.  In round robin pool play, a game can finish in a draw as no penalty shoot-out will follow. In the promotion round, intermediate round, quarter finals, semi finals, 3rd place or final, a game that ends in a draw will be followed by a penalty shoot-out (see \cref{sec:penalty_shoot-out}).
+The clock stops during stoppages of play (such as ready and set state after goals) from the knock-out competition onwards.  In round-robin pool play, a game may finish in a draw and there will be no penalty shoot-out, like in all other games (see Section~\ref{sec:penalty_shoot-out}).
```

Comment:
To which category do promotion games belong?

Reply:
I think this goes with the discussion above → see there.